tools/cli/wskadmin

```diff
@@ -86,6 +86,7 @@ def parseArgs():
 
     subcmd = subparser.add_parser('create', help='create a user and show authorization key')
     subcmd.add_argument('subject', help='the subject to create')
+    subcmd.add_argument('-u', '--auth', help='the uuid:key to initialize the subject authorization key with')
```

Comment:
stashed in this commit but not related to active ack bug - allow wskadmin to create user with a given key. validate the key is a uuid:secrete pair that conforms.